pkg/backup/item_backupper.go

```diff
@@ -105,21 +95,18 @@ type ItemBackupper interface {
 }
 
 type defaultItemBackupper struct {
-	backup                *api.Backup
-	namespaces            *collections.IncludesExcludes
-	resources             *collections.IncludesExcludes
+	backup                *Request
```

Comment:
If we are creating a new struct to hold a backup + the context for creation, I think it would be good to distinguish that here. This is a bit overloaded and I think it would be better as backupRequest or request.